apps/extension/src/ui/hooks/ledger/common.ts

```diff
@@ -49,22 +49,16 @@ export const ledgerNetworks = [
     genesisHash: "0x97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21",
     label: "Nodle",
   },
-  // commented because statemine accounts override kusama accounts when imported
-  // {
-  //   name: "statemine",
-  //   genesisHash: "0x48239ef607d7928874027a43a67689209727dfb3d3dc5e5b03a39bdc2eda771a",
-  // },
-  // commented because statemine accounts override polkadot accounts when imported
-  // {
-  //   name: "statemint",
-  //   genesisHash: "0x68d56f15f85d3136970ec16946040bc1752654e906147f7e43e9d539d7c3de2f",
-  // },
-  // Removed because no longer maintained https://support.ledger.com/hc/en-us/articles/11158925739677-Centrifuge-CFG-
-  // {
-  //   name: "centrifuge",
-  //   genesisHash: "0xb3db41421702df9a7fcac62b53ffeac85f7853cc4e689e0b93aeb3db18c09d82",
-  //   label: "Centrifuge",
-  // },
+  {
+    name: "statemine",
+    genesisHash: "0x48239ef607d7928874027a43a67689209727dfb3d3dc5e5b03a39bdc2eda771a",
+    label: "Statemine", // current name of the app in Ledger Live and on the device, hasn't been renamed to Kusama Asset Hub yet.
+  },
+  {
+    name: "statemint",
+    genesisHash: "0x68d56f15f85d3136970ec16946040bc1752654e906147f7e43e9d539d7c3de2f",
+    label: "Statemint", // current name of the app in Ledger Live and on the device, hasn't been renamed to Polkadot Asset Hub yet.
```

Comment:
Kept it like that because on every screens that this is displayed, it will already be stated that we're interacting with Polkadot Asset Hub.
However it's important for the user to launch the ledger app called "Statemint" on his device. 
Bit concerned about the length, it could overflow in our 2 lines messages boxes

Still want me to change ?

Reply:
additionally, in some case we display the app name followed by the associated token symbol between parenthesis. so if we were to do that, it would display "Stamint (Polkadot Asset Hub) (DOT)

Reply:
OK. In that case, can we change the name manually only in the dropdown where the Ledger app is first chosen?

Reply:
ok, done